OpenProblemLibrary/UBC/MECH/MECH2/Dynamics/Mech2_2016-2017/UBC-DYN-16-009.pg

```diff
@@ -67,9 +67,9 @@ BEGIN_PGML
 The thin plate [`ABCD`] of mass, [`[$m]`] [`kg`] is held in the position shown by the wire [`BH`] and two links [`AE`] and [`DF`]. Neglecting the mass of the links, determine the magnitude of the acceleration of the plate immediately after the wire [`BH`] has been cut.  
 
 
-[`a_{BH}=`][____]{$Ans} [`m/s^2`].
+[`a_{ABCD}=`][____]{$Ans} [`m/s^2`].
```

Comment:
Hi @drdrew42 , the problems in this PR are also linked to a homework set that we have given to students. It would be great if you could approve these as well, thanks for your help with this.

Reply:
Sorry, I somehow missed this comment in the stack of emails from GitHub.